src/libsync/vfs/off/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_library(owncloud_vfs_off MODULE
+    vfs_off.cpp
+)
+
+set_target_properties(owncloud_vfs_off PROPERTIES OUTPUT_NAME "${synclib_NAME}_vfs_off")
+
+target_link_libraries(owncloud_vfs_off
+    libsync
```

Comment:
Does it make sense to specify the linkage type?

Reply:
Hmm yes but no one should ever try to link to the plugin...